variables.tf

```diff
@@ -72,6 +72,32 @@ variable "alb_5xx_count_evaluation_periods" {
 
 
 
+variable "alb_5xx_rate" {
+  type        = string
+  description = "5XX responses from the ALB relativ to requests (static|off)."
+  default     = "off"
+}
+
+variable "alb_5xx_rate_threshold" {
+  type        = number
+  description = "The maximum rate (in %) of 5XX responses from the ALB (0-100)."
+  default     = 5
+}
+
+variable "alb_5xx_rate_period" {
+  type        = number
+  description = "The period in seconds over which the specified statistic is applied (<= 86400 and multiple of 60)."
+  default     = 60
```

Comment:
I'd suggest a higher default for workloads with low/medium traffic. At least `300`.